<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow2</class>
 <widget class="QDialog" name="loginwindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;User Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>100</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="LoginButton">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Guest">
       <property name="text">
        <string>Continue as a Guest</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="gameOn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Let's play a game.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LoginButton</sender>
   <signal>clicked()</signal>
   <receiver>loginwindow2</receiver>
   <slot>onPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Guest</sender>
   <signal>clicked()</signal>
   <receiver>loginwindow2</receiver>
   <slot>guestCheckout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gameOn</sender>
   <signal>clicked()</signal>
   <receiver>loginwindow2</receiver>
   <slot>gameOn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPushButtonClicked()</slot>
  <slot>guestCheckout()</slot>
  <slot>gameOn()</slot>
 </slots>
</ui>
